packages/components/tables/_tables.scss

```diff
@@ -71,3 +71,82 @@
 .nhsuk-table__cell--numeric {
   text-align: right;
 }
+
+$mobile: 768px;
```

Comment:
To avoid added another breakpoint could this not use the `desktop` breakpoint which is 769px?